### DISCUSSION TITLE
Sentry crashes complete website on PHP 7.4

### DISCUSSION THREAD

**Discussion by JE4GLE** [getsentry/sentry-php#1286] [Q&A] [answered]
### Environment

How do you use Sentry?
Sentry SaaS (sentry.io)

Which SDK and version?
sentry/sdk                       3.1.1
sentry/sentry                  3.3.7

My personal composer require contains:
    "php": ">=7.4",
    "sentry/sdk": "^3.0.0"

### Steps to Reproduce

1. Updated Sentry using `composer update`
2. Uploaded vendor directory to my PHP 7.4 webserver
3. My website instantly displays the error message mentioned below (Case A)
4. I updated my composer again using `composer update --ignore-platform-reqs`
5. Uploaded vendor directory again
6. Now, nothing is shown (Case B), even with `error_reporting(true);` and `ini_set('display_errors', true);`
7. I remove the Sentry initialization and now my website works again

### Expected Result

My Website to display

### Actual Result

Case A: Just the message: `Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.0.2".`

Case B: No response, just white

### Comment

According to my steps, I think the error is in my configuration or Sentry itself. Please tell me, if you need some additional information. Disabling Sentry is just a temporary fix for me and I want to have it back up running as soon as possible :)

**Comment by stayallive** [comment, marked as answer]
This is a composer error and has nothing to do with us or our SDK because we indeed support PHP 7.4.

You have probably ran `composer require sentry/sdk` on a machine that has PHP 8 installed and are running that site under PHP 7.4 triggering the error.

Make sure you run composer under the correct PHP version and the issue should be fixed.
